### PR TITLE
Fix Stud.IP portal of the University of Passau

### PR DESCRIPTION
**Pull request by ThexXTURBOXx**
This fixes the following artifacts:
grafik

After applying the patch, it looks like this:
grafik

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -15998,6 +15998,15 @@ paper-radio-button
 
 ================================
 
+studip.uni-passau.de
+
+CSS
+#layout_container > div {
+  background: 0;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:16005]: This isn't valid CSS. Maybe try:
```suggestion
  background-image: none;
```

Comment (ThexXTURBOXx) [reply src/config/dynamic-theme-fixes.config:16005]: It actually is valid for Stud.IP itself as can be seen here (this is the standard configuration which gets overriden by Dark Reader):
grafik

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:16005]: It's preferred to avoid such syntax.

**Review by Gusted — APPROVED** [review]

**Comment by ThexXTURBOXx** [comment]
@Gusted In that case, I have applied your suggested change. Thanks for the information!

**Pull request merged**